src/services/subject.js

```diff
@@ -0,0 +1,30 @@
+import { categoriesService } from '#services/category.js'
+import { createError } from '#utils/errorsHelper.js'
+import { error } from '#consts/validationError.js'
+import Subject from '#models/subject.js'
+import mongoose from 'mongoose'
+
+const { CATEGORY_NOT_FOUND, SUBJECT_NOT_FOUND, INVALID_ID } = error
+
+export const subjectsService = {
+  getAllSubjects: async () => {
+    const subjects = await Subject.find()
+    if (!subjects.length === 0) throw createError(404, SUBJECT_NOT_FOUND)
+
+    return subjects
+  },
+
+  createSubject: async (subjectData) => {
```

Comment:
What about subject duplication? Should we first check if the subject already exists?